recipe/meta.yaml

```diff
@@ -13,30 +13,32 @@ source:
     - tzdata-location.patch
 
 build:
-  number: 6
-  skip: true  # [py>39]
-  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 7
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"  # [py<39]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - {{ compiler('c') }}
+    - python                                 # [py<39 and build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [py<39 and build_platform != target_platform]
+    - {{ compiler('c') }}                    # [py<39]
   host:
-    - backports
+    - backports            # [py<39]
     - python
     - pip
   run:
-    - backports
+    - backports            # [py<39]
     - python
-    - tzdata
+    - tzdata               # [py<39]
```

Comment:
It looks like if the platform is `win` and the interpreter `pypy` it may need `tzdata` irrespective of the Python version.

![image](https://user-images.githubusercontent.com/881019/198876781-306d5057-b84b-423e-91c3-1c6e924506b3.png)

Reply:
Fixed in pypy package now